lib/valkyrie/persistence/postgres/orm_converter.rb

```diff
@@ -6,113 +6,129 @@ def initialize(orm_object)
       @orm_object = orm_object
     end
 
+    # Create a new instance of the class described in attributes[:internal_resource]
+    # and send it all the attributes that @orm_object has
     def convert!
-      @resource ||= attributes[:internal_resource].constantize.new(attributes)
+      @resource ||= resource
```

Comment:
I did this refactor to make the postgres/orm_converter more like the solr/orm_converter which has these additional methods